CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
-## 1.4.20 (Unreleased)
+## 1.4.20 (April 20, 2023)
```

Comment:
```suggestion
## 1.4.21 (Unreleased)

## 1.4.20 (April 20, 2023)
```

I should make this more clear in the instructions.

Reply:
No, this was me being dumb.